keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1188,6 +1192,8 @@ feature documentation to explain this change. In particular, we are going to
 provide a list of example scenarios impacted by this change, including:
 invalid image reference, invalid config map reference.
 
+For the third iteration of Beta (1.28), we plan to fix a bug where the job controller does not wait for the terminating pods to be fully terminated (see Issue: [Jobs create replace pods as soon as Pod is marked for deletion](https://github.com/kubernetes/kubernetes/issues/115844)).
```

Comment:
Please reference here the new section about the new flag.

Reply:
Updated.